src/test/java/de/tum/in/www1/artemis/notification/GroupNotificationServiceTest.java

```diff
@@ -11,9 +11,7 @@
 import java.util.List;
 import java.util.Set;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Since we do not need more than the three previous imports, we should not combine them with a * import. It seems that this occurred when a new import was added but ultimately not required. It's unfortunate that IntelliJ does not automatically optimize and sort imports by default :/

```suggestion
import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;
```